lib/v2/components/user_info_card.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+import 'package:seeds/v2/components/profile_avatar.dart';
+import 'package:seeds/v2/design/app_theme.dart';
+
+class UserInfoCard extends StatelessWidget {
```

Comment:
This widget is already created in the global components. 

I think its called UserSearchRow.

Reply:
Fix